src/current/v25.4/read-from-standby.md

```diff
@@ -0,0 +1,100 @@
+
+The standby cluster's ReaderVC has its own system tables and cluster settings. The ReaderVC replicates a subset of system tables, including **Users** and **Roles**, from the AppVC, so that existing primary users can authenticate. Other system tables and cluster settings are set to defaults in the ReaderVC.
+
+In the event of failover, the ReaderVC is destroyed.
+
+## Use the read from standby feature
+### Before you begin
+
+Prior to setting up read from standby, ensure that:
+
+- you have already configured PCR between a _primary_ cluster and a _standby_ cluster. For information on configuring PCR, refer to [Set Up Physical Cluster Replication]({% link {{ page.version.version }}/set-up-physical-cluster-replication.md %}).
+- your CockroachDB version is v24.3 or later. The `read from standby` option is not supported in earlier versions.
+
+### Start a PCR stream with read from standby
+
+To start a PCR stream that allows read access to the standby cluster, use the `CREATE VIRTUAL CLUSTER ... REPLICATION` statement with the `READ VIRTUAL CLUSTER` option:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE VIRTUAL CLUSTER main FROM REPLICATION OF main ON 'postgresql://{connection string to primary}' WITH READ VIRTUAL CLUSTER;
+~~~
+
+### Add read from standby to a PCR stream
+
+To add read from standby capabilities to an existing PCR stream, use the `ALTER VIRTUAL CLUSTER` statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER VIRTUAL CLUSTER main SET REPLICATION READ VIRTUAL CLUSTER;
+~~~
+
+### Check the status of your reader virtual cluster
+
+To confirm that your reader virtual cluster is active:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SHOW VIRTUAL CLUSTERS;
+~~~
+
+The output shows a `standby-readonly` virtual cluster in addition to the primary and standby clusters:
+
+~~~
+  id |       name       | data_state  | service_mode
+-----+------------------+-------------+---------------
+   1 | system           | ready       | shared
+   3 | standby          | replicating | none
+   4 | standby-readonly | ready       | shared
+~~~
+
+### Run read-only queries on the standby cluster
+
+Once you have created a reader virtual cluster on the standby cluster, you can connect to it and run read (`SELECT`) queries. For example:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT COUNT(*) FROM customers;
+SELECT region, SUM(amount) FROM orders GROUP BY region;
+~~~
+
+The results of queries on the standby cluster reflect the state of the primary cluster as of the replicated time.
```

Comment:
nit: i think a more accurate way to say this is: "reads are always served at a historical time approaches the replicated time." (its not quite the replicated time currently)